README.md

```diff
@@ -63,5 +63,29 @@ make create-lambda-ebs-backup
 ### Tagging Volumes for backup
 
 In order to backup a volume, you must opt-in by setting a tag key:value pair on
-the volume. By default, this is `lambda-ebs-backup:true`. This tells the lambda
-function that we are to create a snapshot of the volume.
+the volume. By default, this is `lambda-ebs-backup/backup:true`. This tells the
+lambda function that we are to create a snapshot of the volume.
+
```

Comment:
Might be useful to document the tag for reboots here as well: `lambda-ebs-backup/reboot-on-image`

Reply:
Will do.